alphastats/gui/pages/06_LLM.py

```diff
@@ -118,17 +126,35 @@ def llm_config():
     with c11:
         st.write("Upregulated genes")
```

Comment:
this will stay `genes`, right, like the annotation in the volcano will be? then maybe show in the "uniprot preview dropdown" also the gene name (otherwise there's no connection as of now)